indexer/selector.go

```diff
@@ -53,7 +56,7 @@ func (s *selectorBlock) Text(el *goquery.Selection) (string, error) {
 			WithFields(logrus.Fields{"case": s.Case}).
 			Debugf("Applying case to selection")
 		for pattern, value := range s.Case {
-			if el.Is(pattern) {
+			if el.Is(pattern) || el.Has(pattern).Length() >= 1 {
```

Comment:
If you don't like it I can revert it, then the selector in the case would have to be surrounded with ":has()"